shortcodes/team-activity.php

```diff
@@ -52,8 +52,20 @@ function ghactivity_team_shortcode( $atts ) {
 	 */
 	$repo = apply_filters( 'ghactivity_team_report_repo_to_monitor', '' );
 
-	$action_count = GHActivity_Calls::get_summary_counts( $date_start, $date_end, $team, $repo, false );
+	$action_count = GHActivity_Calls::get_summary_counts( $date_start, $date_end, $team, $repo );
 
+	return team_activity_markup( $action_count, $team );
+}
+
+/**
+ * Generates HTML for team_activity shortcode
+ *
+ * @since x.x.x
```

Comment:
You can use 2.0.0 here; I will release 2.0.0 to the W.org repo once this is all working.